### PR TITLE
fix Feign encoders

### PR DESCRIPTION
**Pull request by wivern**

### CODE REVIEW

**Comment by acumarav** [comment]
I have tested login in DataNode - works as expected

File: pom.xml [hunk 1]
```diff
@@ -172,22 +173,27 @@
...
         </dependency>
         <dependency>
             <groupId>io.github.openfeign.form</groupId>
             <artifactId>feign-form</artifactId>
-            <version>2.2.1</version>
+            <version>3.8.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.github.openfeign.form</groupId>
+            <artifactId>feign-form-spring</artifactId>
```
Comment (pavgra) [inline pom.xml]: Where is it used?

Comment (acumarav) [reply pom.xml]: There is not direct usages, i my local environment to make develop branch working I do not use it.
So If there is not some indirect impact - I would suggest to remove it too

Comment (wivern) [reply pom.xml]: @acumarav Could you check if login remains working w/o `feign-form-spring`?
If it wouldn't be broken then we could eliminate `feign-form-spring` as unnecessary.

Comment (acumarav) [reply pom.xml]: @wivern - I am able to login without this dependency, but I see some error at token refresh in both cases with and without that dependency

Comment (wivern) [reply pom.xml]: I believe token refresh is not about feign. Let's remove redundant dependency.

Comment (wivern) [reply pom.xml]: @acumarav thank you for test, and also could you check import of Atlas entities?

**Pull request merged**